### ISSUE TITLE
JSSS traning sample

### ISSUE THREAD

**Issue by ycat3** [kan-bayashi/ParallelWaveGAN#222]
Thank you for the JSSS recipe.
Generated voice sample is here.
https://jweb.asia/26-it/ai/58-jsss-parallel-wavegan.html

**Comment by kan-bayashi** [comment]
Thank you for sharing example.
Your text2mel model (Tacotron2) sounds like JSUT.
Did you use JSSS for tacotorn2 training?
If you use JSUT model (i.e., text2mel and mel2wav are matched), the quality will be improved.

**Comment by ycat3** [comment]
Thank you for the advice.
I used Tacotron2 pretrained model, could be JSUT model.
Anyway JSSS and JSUT corpus are almost identical, probably no significant quality difference.

**Comment by kan-bayashi** [comment]
> Anyway JSSS and JSUT corpus are almost identical, probably no significant quality difference.

The content is almost the same but the problem here is the speaker's difference (i.e., the speaker of the training data).
Maybe you can check the quality difference with this demo and your trained JSSS model.
https://colab.research.google.com/github/espnet/notebook/blob/master/espnet2_tts_realtime_demo.ipynb

Anyway, if you just want to check the behavior, current setting is OK.
If you are interested in the further improvement of the TTS model, try the recent advanced text2mel (e.g. fastspeech2) with the matched PWG :)

**Comment by ycat3** [comment]
Thank you.
I will try.

**Comment by ycat3** [comment]
I have downloaded exp/tts_train_conformer_fastspeech2_raw_phn_jaconv_pyopenjtalk/train.loss.ave_5best.pth
However model.json file is missing when running ja-tts-inference.
Should I train the model from scratch ?

**Comment by kan-bayashi** [comment]
What is `ja-tts-inference`?

**Comment by kan-bayashi** [comment]
Maybe you mention about ESPnet1-based script.
Now we made ESPnet2 to make it more flexible.
The inference procedure is much simpler. See the notebook.

**Comment by ycat3** [comment]
I am using modified run-ja-tts-inference.py script from https://github.com/syoyo/espnet-tts.git.
I will look into the notebook.

**Comment by ycat3** [comment]
Locally modified espnet2_tts_demo.ipynb version is working well with jsss pretrained model.
However, I am using "kan-bayashi/jsut_fastspeech2" tag since "kan-bayashi/jsss_fastspeech2" tag is missing.
Is this OK ?
tag = "kan-bayashi/jsut_fastspeech2"
#tag = "kan-bayashi/jsss_fastspeech2"

**Comment by kan-bayashi** [comment]
Sorry, I've not yet uploaded JSSS fastspeech2.
But you can confirm the mismatch effect by comparing 
- jsut_fastspeech2 + jsut_pwg
- jsut_fastspeech2 + your trained jsss_pwg

**Comment by kan-bayashi** [comment]
I will close this issue.
